### PR TITLE
Bring drake up to date

### PR DESCRIPTION
**Pull request by tkoolen**
Builds with https://github.com/tkoolen/drake/tree/tk-dont-flatten-include-hierarchy.
(pull requested in https://github.com/RobotLocomotion/drake/pull/1612)

### CODE REVIEW

File: src/app/ddDrakeModel.cpp [hunk 1]
```diff
@@ -835,17 +802,14 @@ URDFRigidBodyManipulatorVTK::Ptr loadVTKModelFromFile(const string &urdf_filenam
...
 
     boost::filesystem::path mypath(urdf_filename);
     if (!mypath.empty() && mypath.has_parent_path())  
       pathname = mypath.parent_path().native();
 
-    if (!model->addRobotFromURDFString(xml_string, PackageSearchPaths, pathname))
-    {
-      return URDFRigidBodyManipulatorVTK::Ptr();
-    }
+    model->addRobotFromURDFString(xml_string, PackageSearchPaths, pathname);
```
Comment (patmarion) [inline src/app/ddDrakeModel.cpp]: does this method not return an error code anymore?

**Comment by patmarion** [comment]
Looks good, thanks!

File: src/app/ddDrakeModel.cpp (hunk 1 above)
Comment (tkoolen) [inline src/app/ddDrakeModel.cpp]: Nope, void.

Comment (tkoolen) [inline src/app/ddDrakeModel.cpp]: And I didn't want to catch just any runtime_error.

Comment (patmarion) [inline src/app/ddDrakeModel.cpp]: Ok.  This method is typically called from Python which can catch the exception.

**Comment by patmarion** [comment]
I'm happy to merge this now.  Agreed?  Once this is merged, the openhumanoids PR can be updated with a new sha1 for director.  I will not merge any more PRs to director after this one until the openhumanoids PR has been merged.

**Comment by tkoolen** [comment]
Yep, great.

**Pull request merged**